app/assets/javascripts/hikes/hikes_search.js

```diff
@@ -51,7 +51,12 @@ function addDataTable() {
         { "width": "12.5%" },
         { "width": "12.5%" },
         { "width": "15%" }
-      ]
+      ],
+      "fnDrawCallback": function(oSettings) {
+        if ($('#searched_hikes tr').length < 26) {
+            $('.dataTables_paginate').hide();
```

Comment:
because the 25 limit is set above, its better to set it somewhere in one place and call DISPLAY_LIMIT + 1 for the check, rather than hardcoding 25 above and 26 here.